eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -62,6 +62,9 @@ object Channel {
   val MIN_CLTV_EXPIRY = 9L
   val MAX_CLTV_EXPIRY = 7 * 144L // one week
 
+  //  since BOLTv1.1
+  val MAX_MAIN_CLTV = 2016L
```

Comment:
```suggestion
  val MAX_TO_SELF_DELAY = 2016L
```

This is the "official" name of this parameter in the BOLTs, I've never seen a reference to a main cltv.

Reply:
Also, `toSelfDelay` should be an `Int`, not a `Long`